scripts/adeli/adeli.js

```diff
@@ -0,0 +1,56 @@
+const { exec } = require('child_process');
+
+const execShellCommand = (cmd) => new Promise((resolve, reject) => {
+  exec(cmd, (error, stdout, stderr) => {
+    if (error) {
+      console.warn(error);
+    }
+    resolve(stdout || stderr);
+  });
+});
+
+const match = (input, result) => {
+  if (!input) return true;
+  const inputWithoutSpace = input.replace(' ', '');
+  return inputWithoutSpace.localeCompare(result, undefined, { sensitivity: 'base' }) === 0;
```

Comment:
complexe... why not a simple === ?

Reply:
comparing like this will return true if input = "Firstname " and result = "FIRSTNAME"